diff --git a/dist/ui/main.d.ts b/dist/ui/main.d.ts
index 40a06bf13912ca8e3e5d9db61dccea3278d951d6..aed149837d1c7ae66685cd00f959ce501f5ea09f 100644
--- a/dist/ui/main.d.ts
+++ b/dist/ui/main.d.ts
@@ -52,7 +52,10 @@ export declare const componentManager: ComponentManager;
 
 export declare const extensionManager: ExtensionManager;
 
-export declare const panel: Panel;
+export declare const panel: Panel & {
+    _leftBox: St.BoxLayout
+    _rightBox: St.BoxLayout
+};
 
 export declare const overview: any;
 
diff --git a/src/ui/main.d.ts b/src/ui/main.d.ts
index 40a06bf13912ca8e3e5d9db61dccea3278d951d6..aed149837d1c7ae66685cd00f959ce501f5ea09f 100644
--- a/src/ui/main.d.ts
+++ b/src/ui/main.d.ts
@@ -52,7 +52,10 @@ export declare const componentManager: ComponentManager;
 
 export declare const extensionManager: ExtensionManager;
 
-export declare const panel: Panel;
+export declare const panel: Panel & {
+    _leftBox: St.BoxLayout
+    _rightBox: St.BoxLayout
+};
 
 export declare const overview: any;
 
