diff --git a/dist/ui/main.d.ts b/dist/ui/main.d.ts
index 8287446cbedbdadf0ebe219d2747d7d1acc621ba..1e84a235d14364bbdc9e9a0d8144c6a13a324438 100644
--- a/dist/ui/main.d.ts
+++ b/dist/ui/main.d.ts
@@ -52,7 +52,10 @@ export declare const componentManager: ComponentManager;
 
 export declare const extensionManager: ExtensionManager;
 
-export declare const panel: Panel;
+export declare const panel: Panel & {
+  _leftBox: St.BoxLayout
+  _rightBox: St.BoxLayout
+};
 
 export declare const overview: any;
 
diff --git a/src/ui/main.d.ts b/src/ui/main.d.ts
index 8287446cbedbdadf0ebe219d2747d7d1acc621ba..1e84a235d14364bbdc9e9a0d8144c6a13a324438 100644
--- a/src/ui/main.d.ts
+++ b/src/ui/main.d.ts
@@ -52,7 +52,10 @@ export declare const componentManager: ComponentManager;
 
 export declare const extensionManager: ExtensionManager;
 
-export declare const panel: Panel;
+export declare const panel: Panel & {
+  _leftBox: St.BoxLayout
+  _rightBox: St.BoxLayout
+};
 
 export declare const overview: any;
 
