diff --git a/dist/ui/main.d.ts b/dist/ui/main.d.ts
index a63a4217e08ba1d3898ba0bf419e1fb1c714b339..ffd5d001bca6a5f1a7fe1519212dd44891b6cb87 100644
--- a/dist/ui/main.d.ts
+++ b/dist/ui/main.d.ts
@@ -48,12 +48,14 @@ import { WindowManager } from './windowManager.js';
 // import * as Config from '../misc/config.js'
 // import * as Util from '../misc/util.js'
 
-
 export declare const componentManager: ComponentManager;
 
 export declare const extensionManager: ExtensionManager;
 
-export declare const panel: Panel;
+export declare const panel: Panel & {
+  _leftBox: St.BoxLayout
+  _rightBox: St.BoxLayout
+};
 
 export declare const overview: any;
 
diff --git a/src/ui/main.d.ts b/src/ui/main.d.ts
index a63a4217e08ba1d3898ba0bf419e1fb1c714b339..9de042059df4f753963cc0437f567257e3198ae1 100644
--- a/src/ui/main.d.ts
+++ b/src/ui/main.d.ts
@@ -53,7 +53,10 @@ export declare const componentManager: ComponentManager;
 
 export declare const extensionManager: ExtensionManager;
 
-export declare const panel: Panel;
+export declare const panel: Panel & {
+  _leftBox: St.BoxLayout
+  _rightBox: St.BoxLayout
+};
 
 export declare const overview: any;
 
